pkgs/development/python-modules/pythonix/default.nix

```diff
@@ -18,6 +18,13 @@ buildPythonPackage rec {
 
   buildInputs = [ nix boost ];
 
+  postInstall = ''
+    # XXX why does pythonImportsCheck not set this?
+    export PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH
```

Comment:
@FRidh @jonringer Why is this line necessary? This build install a python modules with meson. Is there something else necessary. The python module is a shared-library only, no python source code.

Reply:
Ah, I think I figured it out. The pythonpath is typically set by the pipInstallHook or eggInstallHook. So when we use e.g. a cmake or meson install the package isn't added to the pythonpath. I like your approach much better, so I just updated the comment

Reply:
May want to create a `pyproject.toml` for the project that uses `mesonpep517` as build backend. Then the backend will use meson to build a wheel and we install the wheel using pip.
https://github.com/FRidh/mesonpep517examples

Reply:
I need to wait for upstream to fix this bug so: https://gitlab.com/thiblahute/mesonpep517/-/issues/5